components/reports/precipitation/ReportPrecipTable.vue

```diff
@@ -3,7 +3,8 @@
     <table class="table report-table" v-if="reportData">
       <caption>
         Precipitation,
-        <span v-html="place"></span>, 1950&ndash;2099
+        <span v-html="place"></span
```

Comment:
Hey was this file unlinted or is there potentially a lint-battle happening here?